### PR TITLE
v2.3.1: License Update

### PR DESCRIPTION
**Pull request by mrkaye97**
Getting rid of `xtable` dep here and switching over to MIT

@andrie Does this change look okay to you? Seems like a reasonable way to go assuming I'm not missing anything big here. got rid of `xtable` because it's GPL, which AFAIK should free me up to use MIT

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,2 +1,2 @@
-YEAR: 2014
-COPYRIGHT HOLDER: Bob Rudis
+YEAR: 2021
+COPYRIGHT HOLDER: Matt Kaye
```
Comment (andrie) [inline LICENSE]: This should probably include Bob Rudis as well as yourself.

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# MIT License
+
+Copyright (c) 2021 Matt Kaye
```
Comment (andrie) [inline LICENSE.md]: Add Bob Rudis

File: DESCRIPTION [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Package: slackr
 Type: Package
 Title: Send Messages, Images, R Objects and Files to 'Slack' Channels/Users
-Version: 2.3.0
+Version: 2.3.1
 Date: 2021-05-01
```
Comment (andrie) [inline DESCRIPTION]: You don't need a date in the DESCRIPTION any more.  CRAN will add this for you.

**Comment by andrie** [comment]
This looks OK to me, but given that many other people contributed code, and thus still have the copyright, you should probably follow the advice from https://r-pkgs.org/license.html#relicensing :

> Ask every contributor if they’re OK with changing the license. If every contributor is on GitHub, the easiest way to do this is to create an issue where you list all contributors and ask them to confirm that they’re OK with the change. Two examples where the tidyverse team has relicensed code include generics and covr.

**Comment by mrkaye97** [comment]
Awesome, thanks @andrie!

**Comment by mrkaye97** [comment]
cc @hrbrmstr, @yonicd, @jefferis, @qsweber, @xinye1, @raybuhr

As contributors of non-trivial amounts of code to this package, are you all okay with this license update?

**Comment by jefferis** [comment]
@mrkaye97 fine by me to change license.

**Comment by qsweber** [comment]
Also fine by me!

**Comment by yonicd** [comment]
That is good for me. Ty

**Comment by mrkaye97** [comment]
bumping this @hrbrmstr @xinye1

**Comment by mrkaye97** [comment]
@hrbrmstr @xinye1 I'm bumping this one more time. Just give me a :+1: if you think this change is fine. I'll merge on 6/19 if I don't hear anything back, since I strongly believe that moving over to MIT is a good move for the package.

Thanks!

**Comment by hrbrmstr** [comment]
Oh def make any changes you need/want.

**Pull request merged**